CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+### [1.14.1](https://github.com/openfoodfacts/openfoodfacts-dart/compare/v1.14.0...v1.14.1) (2022-03-22)
```

Comment:
@monsieurtanuki, we are in the right line again 🎉

Reply:
@M123-dev Maybe we're lucky, maybe it's the "fix" versions that are OK. We'll see with the next "feat" version.

My bad: I named #417 "refactor" and it is not included here.

Reply:
https://github.com/openfoodfacts/openfoodfacts-dart/pull/406#issuecomment-1069402575 😉